packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -1048,7 +1212,9 @@ function TableCheckboxCell({cell}) {
               stylesOverrides,
               'react-spectrum-Table-cell'
             )
-          )}>
+          )
+        }>
+        {isTableDraggable && <DragButton />}
```

Comment:
Should we have a dedicated cell/column for the drag handle? It feels odd to navigate up or down from a row selection checkbox and to have focus go to the drag handle, rather than the next adjacent checkbox.

Reply:
I think this is a good point. On the other hand, it may be strange to have a cell/column that is entirely empty unless the row is keyboard focused.